docs/tags.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: sub-navigation
+order: 7
+title: Adding tags
+description: Use tags to categorise posts and make it easier for readers to browse content on your site.
+---
+
+The GOV.UK Eleventy Plugin makes it easy to add tags to your website. Follow these instructions to enable this feature.
+
+## Create a tags page
+
+To include a list of all tags used on your website, create a page that uses the `tags` layout:
```

Comment:
As below, might be worth suggesting a filename where this content should be copied to?

```suggestion
To include a list of all tags used on your website, create a page that uses the `tags` layout. Add this content to a `tags.md` file:
```

Reply:
We don’t do this elsewhere… kinda falls into the realms of documenting how Eleventy works.